articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,254 @@
+
+This article will help you migrate your solution from the old implementation to the new one. First, we will see what this migration is about and which use cases are affected. We will continue with reviewing how you can use [scopes](/scopes) to get tokens with different access rights, and then see all the ways you can use to get an Access Token. Finally, we will review the [Account Linking](/link-accounts) process and the changes introduced to it.
+
+## Overview
+
+Some of the [/users](/api/management/v2#!/Users/get_users_by_id) and [/device-credentials](/api/management/v2#!/Device_Credentials/get_device_credentials) endpoints, could be called using ID Tokens. You can now use Access Tokens to access them. The grace period for this migration has already started and will end on the `1st of June, 2018`, when you will no longer be able to use ID Tokens as credentials for the Management API.
+
+## Does this affect me?
+
+You are affected by this migration if you use ID Tokens to call any of the following endpoints:
+
+| **Endpoint** | **Use Case** |
+|-|-|
+| [GET /api/v2/users/{id}](/api/management/v2#!/Users/get_users_by_id) | Retrieve a user's information |
+| [GET /api/v2/users/{id}/enrollments](/api/management/v2#!/Users/get_enrollments) | Retrieve all [Guardian](/multifactor-authentication/guardian) MFA enrollments for a user |
+| [PATCH /api/v2/users/{id}](/api/management/v2#!/Users/patch_users_by_id) | Update a user's information |
+| [DELETE /api/v2/users/{id}/multifactor/{provider}](/api/management/v2#!/Users/delete_multifactor_by_provider) | Delete the [multifactor](/multifactor-authentication) provider settings for a user |
+| [POST /api/v2/device-credentials](/api/management/v2#!/Device_Credentials/post_device_credentials) | Create a public key for a device |
+| [DELETE /api/v2/device-credentials/{id}](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) | Delete a device credential |
+| [POST/api/v2/users/{id}/identities](/api/management/v2#!/Users/post_identities) | [Link user accounts](/link-accounts) from various identity providers |
+| [DELETE /api/v2/users/{id}/identities/{provider}/{user_id}](/api/management/v2#!/Users/delete_provider_by_user_id) | [Unlink user accounts](/link-accounts#unlinking-accounts) |
+
+Note that the last two endpoints are used for [Account Linking](/link-accounts#the-management-api). The changes in this functionality are the following:
+- You can no longer use an ID Token at the `Authorization` header
+- If you already use Access Tokens at the `Authorization` header you don't need to change something in your implementation, but do note that now you can send at the request's body the secondary account's ID Token, instead of the `user_id` (or you can keep using the `user_id`)
```

Comment:
> (or you can keep using the `user_id`)

You can keep using `user_id` with scope `update:users`. Scope `update:current_user` (the equivalent of using `id_token` for auth) won't support using `user_id`.